### PR TITLE
Update ms-cloud-germany-transition-add-devices.md

### PR DESCRIPTION
**Pull request by Ariantor**
Add 1 entry FAQ in the Devices section

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/ms-cloud-germany-transition-add-devices.md [hunk 1]
```diff
@@ -152,6 +152,10 @@ It's critical to your success that you only unregister and re-register your devi
 
 To check whether your devices are registered in the public cloud, you should export and download the list of devices from the Azure AD portal to an Excel spreadsheet. Then, filter the devices that are registered (by using the _registeredTime_ column) after the date when your organization has passed [phase 9 of the migration process](ms-cloud-germany-transition-phases.md#phase-9--10-azure-ad-finalization).
 
+**Do I still need to add the DNS name as stated in this [Create DNS records for Microsoft using Windows-based DNS](https://docs.microsoft.com/en-us/microsoft-365/admin/dns/create-dns-records-using-windows-based-dns?view=o365-worldwide#add-two-cname-records-for-mobile-device-management-mdm-for-microsoft)
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ms-cloud-germany-transition-add-devices.md:155]:
```suggestion
**Do I still need to add the DNS name as stated in [Create DNS records for Microsoft using Windows-based DNS](/microsoft-365/admin/dns/create-dns-records-using-windows-based-dns?view=o365-worldwide#add-two-cname-records-for-mobile-device-management-mdm-for-microsoft)?**
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Ariantor, kindly commit my suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
Gentle reminder @Ariantor

**Pull request merged**